build-master.config

```diff
@@ -0,0 +1,9 @@
+[component]
+component=iof
+
+[commit_versions]
+CART = 0b6f9a14ca4bf6b045ecf078c64e812c669996f6
+FUSE = 3e2fcf3a630e575bc420df254525834504dc01b7
```

Comment:
Shouldn't/couldn't these both be "master" since the build/test is always supposed to be using master?

Reply:
See my comment above, essentially yes, but there are API changes to both CaRT and FUSE that means the current code won't run with actual master so this is as close as we can get.  The next step will be to move IOF forward past the API change, so that IOF can be built against both blessed versions and master.